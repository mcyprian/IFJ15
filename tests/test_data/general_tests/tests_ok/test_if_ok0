int main()
{
	int a=5;
	if(a>3)
	{
		cout << a <<" is bigger than 3\n"; 
	}
	else
	{
		cout << a <<" is not bigger than 3\n";
	}

	if (a==4)
	{ 
		cout << a <<" is equal 4\n"; 
	}
	else
	{
		cout << a <<" is not equal 4\n"; 
	}
	
	return 0;
}