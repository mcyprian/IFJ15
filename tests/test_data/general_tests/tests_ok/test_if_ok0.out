5 is bigger than 3
5 is not equal 4
